autoscheduler/frontend/src/.eslintrc.js

```diff
@@ -23,10 +23,10 @@ module.exports = {
     '@typescript-eslint',
     'import',
   ],
-  rules: {
-    "import/no-extraneous-dependencies": "off",
-    "react/jsx-filename-extension": [1, {"extensions": [".jsx", ".tsx"]}],
```

Comment:
I believe we still need line 28, as I'm currently getting `JSX not allowed in files with extension '.tsx'` from `eslint(react/jsx-filename-extension)`

Reply:
Correct, I had made that error go away by accidentally overwriting the Airbnb rules, but I will add back in the tsx rule